oak-run-commons/src/main/java/org/apache/jackrabbit/oak/run/cli/CommonOptions.java

```diff
@@ -36,13 +36,15 @@ public class CommonOptions implements OptionsBean {
     private final OptionSpec<String> nonOption;
     private final OptionSpec<Void> metrics;
     private final OptionSpec<Void> segment;
+    private final OptionSpec<Void> continueMissingDep;
     private OptionSet options;
 
     public CommonOptions(OptionParser parser){
         help = parser.acceptsAll(asList("h", "?", "help"), "Show help").forHelp();
         readWriteOption = parser.accepts("read-write", "Connect to repository in read-write mode");
         metrics = parser.accepts("metrics", "Enables metrics based statistics collection");
         segment = parser.accepts("segment", "Use older oak-segment support");
+        continueMissingDep = parser.accepts("continue-missing-tika-dep", "Continue to run when there are missing dependency");
```

Comment:
maybe `ignore-missing-tika-dep` would be a better option ? 

`ignoreMissingTikaDep = parser.accepts("ignore-missing-tika-dep", "Ignore when there are missing tika dependencies and continue to run");`

Reply:
changed.